### PR TITLE
Remove pytest dependency in arm_compute_lib.py

### PR DESCRIPTION
**Pull request by NicolaLancellotti**
This pr removes the unexpected dependency on pytest in python/tvm/relay/op/contrib/arm_compute_lib.py

### CODE REVIEW

**Comment by NicolaLancellotti** [comment]
Please, can you review it? @mbaret @comaniac @d-smirnov

File: python/tvm/relay/op/contrib/arm_compute_lib.py [hunk 1]
```diff
@@ -111,20 +111,38 @@ def convert_conv(attrs, inputs, tinfos, desired_layouts):
...
+
+    op_nn = relay.op.get("nn.conv2d")
+    older_attr_nn = op_nn.get_attr(attr_key)
+    op_nn.reset_attr(attr_key)
+    op_nn.set_attr(attr_key, convert_layout_conv2d(tvm.relay.nn.conv2d))
+
+    op_qnn = relay.op.get("qnn.conv2d")
+    older_attr_qnn = op_qnn.get_attr(attr_key)
+    op_qnn.reset_attr(attr_key)
+    op_qnn.set_attr(attr_key, convert_layout_conv2d(tvm.relay.qnn.op.conv2d))
```
Comment (comaniac) [inline python/tvm/relay/op/contrib/arm_compute_lib.py:125]: Better to use `with` context as `TempOpAttr` does to make sure the changes won't affect other processes. I think it's fine to just copy `TempOpAttr` here to be `ArmComputeLibOpAttrContext` or something like that.

**Review by comaniac — CHANGES_REQUESTED** [review]
Also cc @lhutton1

File: python/tvm/relay/op/contrib/arm_compute_lib.py (hunk 1 above)
Comment (lhutton1) [reply python/tvm/relay/op/contrib/arm_compute_lib.py:125]: Agreed, I think this would make it more readable as well, otherwise this looks good to me!

Comment (NicolaLancellotti) [reply python/tvm/relay/op/contrib/arm_compute_lib.py]: I have added `OpAttrContext` (which is a copy of `TempOpAttr`) in `arm_compute_lib.py`.
And I have replaced `TempOpAttr` with `OpAttrContext`.

**Review by lhutton1 — APPROVED** [review]

**Review by comaniac — CHANGES_REQUESTED** [review]
Please rebase again and make sure you didn't change other components (i.e., submodules).

**Review by comaniac — APPROVED** [review]
LGTM

**Comment by comaniac** [comment]
Thanks @NicolaLancellotti @lhutton1

**Pull request merged**